runtime/astar/Cargo.toml

```diff
@@ -79,7 +79,7 @@ try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "p
 
 # Astar pallets
 pallet-block-reward = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-block-reward-2.0.0/polkadot-v0.9.17", default-features = false }
-pallet-collator-selection = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.17", default-features = false }
+pallet-collator-selection = { git = "https://github.com/AstarNetwork/astar-frame", tag = "pallet-collator-selection-3.0.0/polkadot-v0.9.17", default-features = false }
```

Comment:
Sorry, I missed this - this needs spec version update.

Sasa's PR won't change astar, only shibuya.

Reply:
We don’t need to change Astar version, this tag is used for prevent behavior changes